publisher-sdk/src/main/java/com/criteo/publisher/integration/IntegrationRegistry.kt

```diff
@@ -17,8 +17,8 @@
 package com.criteo.publisher.integration
 
 import android.content.SharedPreferences
-import com.criteo.publisher.annotation.OpenForTesting
 import androidx.annotation.VisibleForTesting
+import com.criteo.publisher.annotation.OpenForTesting
```

Comment:
Looks unrelated to its commit

Reply:
Maybe now it seems more related => as test need the class to be opened